packages/app/src/app/pages/NewDashboard/Content/routes/StartSandboxes/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { Element, Text, Button } from '@codesandbox/components';
+import { useQuery } from '@apollo/react-hooks';
+import { useOvermind } from 'app/overmind';
+import {
+  RecentSandboxesQuery,
+  RecentSandboxesQueryVariables,
+  ListPersonalTemplatesQuery,
+  ListPersonalTemplatesQueryVariables,
+} from 'app/graphql/types';
+import { LIST_PERSONAL_TEMPLATES } from 'app/components/CreateNewSandbox/queries';
+import { RECENT_SANDBOXES_CONTENT_QUERY } from '../../../queries';
+import { SandboxCard } from '../../../Components/SandboxCard';
+
+export const StartSandboxes = () => {
+  const {
+    state,
+    actions: { modalOpened },
+  } = useOvermind();
+  const { loading, error, data } = useQuery<
+    RecentSandboxesQuery,
+    RecentSandboxesQueryVariables
+  >(RECENT_SANDBOXES_CONTENT_QUERY, {
+    variables: {
+      orderField: state.dashboard.orderBy.field,
+      // @ts-ignore
+      orderDirection: state.dashboard.orderBy.order.toUpperCase(),
+    },
+  });
+
+  const {
+    data: templatesData,
+    error: templatesError,
+    loading: templatesLoading,
+  } = useQuery<ListPersonalTemplatesQuery, ListPersonalTemplatesQueryVariables>(
+    LIST_PERSONAL_TEMPLATES,
+    {
+      variables: {},
+      fetchPolicy: 'cache-and-network',
+    }
+  );
+
+  if (error || templatesError) {
+    return <Text>Error</Text>;
+  }
+
+  if (loading || templatesLoading) {
+    return <Text>loading</Text>;
+  }
+
+  const sandboxes = data && data.me && data.me.sandboxes;
+  const templates =
+    templatesData &&
+    templatesData.me &&
+    templatesData.me.recentlyUsedTemplates.slice(0, 4);
```

Comment:
Since we know for sure there's no error or loading state, we know `templatesData.me.recentlyUsedTemplates` is present

```suggestion
  const templates = templatesData.me.recentlyUsedTemplates.slice(0, 4);
```